specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/reservation.json

```diff
@@ -98,63 +117,197 @@
             "$ref": "./types.json#/parameters/billingAccountName"
           },
           {
-            "description": "May be used to filter by reservation properties. The filter supports 'eq', 'or', and 'and'. It does not currently support 'ne', 'gt', 'le', 'ge', or 'not'.",
-            "in": "query",
-            "name": "$filter",
-            "required": false,
-            "type": "string"
+            "$ref": "#/parameters/ReservationOrderIdParameter"
           },
           {
-            "description": "May be used to sort order by reservation properties.",
-            "in": "query",
-            "name": "$orderby",
-            "required": false,
-            "type": "string"
+            "$ref": "#/parameters/ReservationIdParameter"
           },
           {
-            "description": "To indicate whether to refresh the roll up counts of the reservations group by provisioning states",
-            "in": "query",
-            "name": "refreshSummary",
-            "required": false,
-            "type": "string"
+            "$ref": "#/parameters/ExpandParameter"
+          }
+        ],
+        "tags": [
+          "Reservation"
+        ],
+        "responses": {
+          "200": {
+            "description": "Get `Reservation` details.",
+            "schema": {
+              "$ref": "#/definitions/Reservation"
+            }
           },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/Microsoft.Capacity/reservationOrders/{reservationOrderId}/reservations": {
+      "get": {
```

Comment:
Just curious, does this endpoint support OData filter, search, orderBy, etc.? We've been documenting those parameters in the BillingRP side.

Note that ARM team doesn't want the `$` as part of the path. See [guidelines here](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options).

Reply:
Discussed offline, we should take it to ARM